Library/Formula/pv.rb

```diff
@@ -10,16 +10,24 @@ class Pv < Formula
     sha256 "f343368e557cb1c86173bd0c62143b34834e2b825b1a188ac2a37c23d0c685dd" => :mountain_lion
   end
 
-  depends_on "gettext"
+  option "with-nls", "Build with Native Language Support"
+
+  depends_on "gettext" if build.with? "nls"
```

Comment:
Ideally we'll add a DSL hack at some point that lets you alias `with-gettext` as `with-nls`